nightingale-server-staging.yaml

```diff
@@ -0,0 +1,196 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: nightingale-staging
+---
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: default-mem-limits
+  namespace: nightingale-staging
+spec:
+  limits:
+  - default:
+      memory: 512Mi
+    defaultRequest:
+      memory: 512Mi
+    type: Container
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nightingale-staging
+  namespace: nightingale-staging
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.nightingale-server: glob:master-*
+    flux.weave.works/tag.migrate: glob:master-*
+    reloader.stakater.com/auto: "true"
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: nightingale-staging
+  template:
+    metadata:
+      labels:
+        app: nightingale-staging
+    spec:
+      imagePullSecrets:
+      - name: kube-pull
+      containers:
+      - name: nightingale-staging
+        image: harbor.nightingale.revelry.org/library/nightingale-server:develop-c030c6e
```

Comment:
```
harbor.nightingale.revelry.org/library/nightingale-server:master
```

NOTE: After we create the `develop` branch and it pushes a docker image to harbor, we'll need to switch this out to `:develop` and switch the globs to `glob:develop-*`. (But not in this PR.)